app/controllers/user_controller.rb

```diff
@@ -0,0 +1,11 @@
+class UserController < ApplicationController
+  layout "admin"
+  def admin; end
```

Comment:
method này để làm gì thế

Reply:
a cho em hỏi là khi viết thế này trong routes
 get "/admin", to: "user#admin" 
thì cái user#admin nghĩa là thế nào anh ơi. cái admin này là 1 cái action thì nghĩa là gì nhỉ a. Có phải là 1 views không ạ.

Reply:
@namanh2610 ừm đúng rồi nó tương ứng với trong controller chú phải có 1 action là `admin` để rails mapping vào nhưng không ai làm thế trừ trường hợp bắt buộc cả.

Hiện vẫn đang trong quá trình training nên các chú cố gắng thiết kế theo `restful` nhé 

Các màn hình của admin thì các chú tạo thêm 1 thư mục trong folder `controller` tên là admin rồi nhét vào thư mục đó các controller liên quan đến quản lý như: `categories_controller`, `users_controller`,.. nhé.

giả giụ như trường hợp này, chú muốn action admin để show ra giao diện chính của admin, nếu giao diện đó chú dự tính sẽ là màn hình dashboard thì set nó là index của controller dashboard nhé.